leanpkg/leanpkg/main.lean

```diff
@@ -149,32 +148,32 @@ configure
 def usage := "
 Usage: leanpkg <command>
 
-configure       download dependencies
-build           download dependencies and build *.olean files
-test            download dependencies and run test files
+configure            download dependencies
+build -- <lean-args> download dependencies and build *.olean files
```

Comment:
I think we should put brackets (the `[]` variety) around `-- <lean-args>` to indicate that the extra arguments are optional.